### PR TITLE
Feature implementation: Commenting system for posts

### PR DESCRIPTION
**Pull request by theagora**
**Feature proposal v2**: https://github.com/agorahub/blog0/issues/3#issuecomment-726799802
- [ ] Blog readers can COMMENT on each post
- `1-category-1-issue` 
- `1-category-1-page`

### CODE REVIEW

**Review by andyof — APPROVED** [review]
Well-done flow: 
- `front page` -> `category comments` -> `new comment` 

Expected flows: 
- `category page` <-> `category comments` 
- ~~`post page` -> `new comment`~~

**Comment by theagora** [comment]
> **Feature proposal v2**: [#3 (comment)](https://github.com/agorahub/blog0/issues/3#issuecomment-726799802)
> - [ ]  Blog readers can COMMENT on each post
> - `1-category-1-issue`
> - `1-category-1-page`

Update: **Feature proposal v2.1**: https://github.com/agorahub/blog0/issues/3#issuecomment-727155350
- [x] `front page` -> `category comments`
- [x] `category comments` -> `new comment`
- [x] `category page` <-> `category comments`

File: initium.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ layout: page
   <div>
     {% for post in category.last %}
     <span class="postdate">{{ post.date | preserve_timezones }}</span>
+    • <span class="author"><a href="{{site.url}}/{{site.baseurl}}/{{post.categories.first}}_comments/?[{{post.title}}]({{site.url}}{{site.baseurl}}/{{post.url}})"><b>💬</b></a></span>
     {% if post.author != null %} • <span class="author">{{ post.author }}</span> {% endif %}
     <h4><a href="{{ site.url }}{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></h4>
     {% endfor %}
   </div>
-  {% endif %}
+  {% endif %}the category comments page
```
Comment (andyof) [inline initium.md:15]: There might be some typo.

Comment (theagora) [reply initium.md]: :+1: got fixed.

**Pull request merged**